react/features/base/conference/functions.ts

```diff
@@ -122,15 +121,15 @@ export function commonUserJoinedHandling(
  * @returns {void}
  */
 export function commonUserLeftHandling(
-        { dispatch }: Object,
-        conference: Object,
-        user: Object) {
+        { dispatch }: IStore,
+        conference: any,
+        user: any) {
     const id = user.getId();
 
     if (user.isHidden()) {
         dispatch(hiddenParticipantLeft(id));
     } else {
-        const isReplaced = user.isReplaced && user.isReplaced();
+        const isReplaced = user?.isReplaced();
```

Comment:
I think this should be `user.isReplaced?.()` since it's the function what might not exist, not the user.